plugins/gnostic-go-generator/renderer.go

```diff
@@ -26,14 +26,16 @@ import (
 
 // Renderer generates code for a surface.Model.
 type Renderer struct {
-	Model   *surface.Model
-	Package string // package name
+	Model             *surface.Model
+	Package           string // package name
+	SortParameterList bool
 }
 
 // NewServiceRenderer creates a renderer.
 func NewServiceRenderer(model *surface.Model) (renderer *Renderer, err error) {
 	renderer = &Renderer{}
 	renderer.Model = model
+	renderer.SortParameterList = false // default
```

Comment:
I'm going to ask you to change existing code to improve the style if you don't mind. Not sure why this is has an `error` in the return type either

```go
return &Renderer{ 
  Model: model,
  SortParams: false,
}, nil
```